Assignments/SCF-Session-2/FCFS/JobScheduler.java

```diff
@@ -0,0 +1,175 @@
+package FCFS;
+
+import java.util.Scanner;
+
+class FirstComeFirstServe
+{
+	
+	int[] findWaitingTime (int processes, int inputTimes[][])
```

Comment:
Access modifiers are missing from all the methods.

Reply:
done